chromadb/utils/embedding_functions.py

```diff
@@ -162,6 +174,20 @@ def __init__(
         self._session.headers.update({"Authorization": f"Bearer {api_key}"})
 
     def __call__(self, texts: Documents) -> Embeddings:
+        """
```

Comment:
In general users shouldn't call EmbeddingFunction's directly (Chroma calls them for you in collections), but I don't mind this extra documentation in case they do want to do that. @HammadB thoughts?

Reply:
I think its nice to have them!